thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyFunctionResource.java

```diff
@@ -19,6 +19,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@Path("thirdeye")
```

Comment:
can it just start with function or anomaly-function then? will be more consistent with the other resources and also with it's own name
Other than that, LGTM